static/js/hotkey.js

```diff
@@ -845,6 +861,10 @@ export function process_hotkey(e, hotkey) {
         return false;
     }
 
+    if (inbox_util.is_visible()) {
+        return false;
+    }
```

Comment:
This is an example of something that should instead be replaced with a better abstraction, ideally. See

https://chat.zulip.org/#narrow/stream/6-frontend/topic/recent_topics_util.2Eis_visible/near/1506369